workflow-job

```diff
@@ -261,10 +261,10 @@ flows["${sha1}"] = {
         node {
             parallel("Building Java 8 ${buildId}": {
                 sh "docker build --tag=${images.jamesCompile} dockerfiles/compilation/java-8"
-                sh "cd dockerfiles/; docker run --rm ${verbose} --tmpfs=/tmp:rw,size=4g --tmpfs=/james-project:rw,size=4g --name=${containers.jamesCompile} ${dockerEnv} ${useMavenCache} -e DOCKER_CONTAINER=${containers.jamesCompile} --volumes-from=${containers.keys} --volumes-from=${containers.gitPublish} --volume=/destination ${images.jamesCompile} ${mergeBranch}"
+                sh "cd dockerfiles/; docker run --rm ${verbose} --tmpfs=/tmp:rw,size=8g --tmpfs=/james-project:rw,size=8g --name=${containers.jamesCompile} ${dockerEnv} ${useMavenCache} -e DOCKER_CONTAINER=${containers.jamesCompile} --volumes-from=${containers.keys} --volumes-from=${containers.gitPublish} --volume=/destination ${images.jamesCompile} ${mergeBranch}"
```

Comment:
I don't agree with this change:
x right now, the build is ok
x 8g would practically limit the builds to 2 on our server (32g), I prefer being conservation about that and keep 4g until we have stability problems